src/adapter/http/src/smart_protocol/phases.rs

```diff
@@ -9,6 +9,7 @@
 
 /////////////////////////////////////////////////////////////////////////////////////////
 
+#[allow(clippy::enum_variant_names)]
```

Comment:
Similarly, debatable. Here we can probably simplify names of the phases.

Reply:
Agreed -- after a quick look at the code, I thought it might affect the communication protocols (backend/frontend) and thought it was beyond the scope of my task. 

We can add an issue to the backlog, what do you think?

Reply:
Just for the record: will be fixed